### ISSUE TITLE
WideDeep

### ISSUE THREAD

**Issue by TrumanDu** [ZiyaoGeng/Recommender-System-with-TF2.0#84]
`outputs = tf.nn.sigmoid(0.5 * wide_out + 0.5 * deep_out)`
这里为何要乘0.5？
